incubator/docker-registry/templates/registry.yaml

```diff
@@ -0,0 +1,79 @@
+{{- $replicas := .Values.replicas }}
+{{- $svcName := .Values.svc_name }}
+{{- $initLoad := .Values.initial_load }}
+{{- $distro := .Values.distro }}
+{{- $type := .Values.type }}
+{{- $branch := .Values.branch }}
+{{- $nodePort := .Values.node_port }}
+{{- $initImage := .Values.init_image }}
+{{- $initImageV := .Values.init_image_version }}
+{{- $registryImage := .Values.registry_image }}
+{{- $registryImageV := .Values.registry_image_version }}
+{{- $tarballURL := .Values.tarball_url }}
+---
```

Comment:
This service requires great flexibility at least t=fir the use case we use it in koala-kubernetes project. On the other hand we do not want to impose complexity on other users, thus providing defaults values. It is required to support both and other use cases.